models/public/mtcnn-p/mtcnn-p.md

```diff
@@ -0,0 +1,44 @@
+# mtcnn-p
+
+## Use Case and High-Level Description
+
+The `mtcnn-p` model is one of the [mtcnn](https://arxiv.org/ftp/arxiv/papers/1604/1604.02878.pdf) group of models designed to perform face detection. Short for "Multi-task Cascaded Convolutional Neural Network", it is implemented using the Caffe framework. The "p" designation indicates that this model is the "proposal" network intended to find the initial set of faces. For details about this family of models, check out the [repository](https://github.com/DuinoDu/mtcnn).
+
+The model input is an image containing the data to be analyzed.
+
+The model output is a blob with a vector containing the first pass of face data. If there are no faces detected, no further processing is needed. Otherwise, you will typically use this output as input to the `mtcnn-r` model.
+
+## Example
+
+## Specification
+
+| Metric            | Value         |
+|-------------------|---------------|
+| Type              | Detection     |
+| GFLOPs            | 3.366         |
+| MParams           | 0.007         |
+| Source framework  | Caffe         |
+
+## Accuracy
+
+## Performance
+
+## Input
+
+Image, shape - `1,3,720,1280`, format is `B,C,H,W`, where:
```

Comment:
format is B,C,**W**,H. It is very important. Otherwise model result will have wrong bounding boxes